sympy/physics/control/tests/test_lti.py

```diff
@@ -1811,6 +1811,37 @@ def test_StateSpace_functions():
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
 
 
+def test_StateSpace_symbolics():
```

Comment:
I don't like the idea of a separate "symbolics" test, why can't we add test each for all methods present following their test.

Reply:
Can you also address this comment in the tests (especially the step function because it's so commonly used). https://github.com/sympy/sympy/pull/26685#discussion_r1646604553

Reply:
> Can you also address this comment in the tests (especially the step function because it's so commonly used). [#26685 (comment)](https://github.com/sympy/sympy/pull/26685#discussion_r1646604553)

I've already included it when I was working on `dsolve()` .
https://github.com/sympy/sympy/blob/0710d784347042c54270fd562a47317607b9be30/sympy/physics/control/tests/test_lti.py#L1744-L1747

Reply:
I'll request a review from Jason as well. He has good ideas on what the control module should be capable of doing !

Reply:
Sure